sections/introduction.tex

```diff
@@ -0,0 +1,61 @@
+More specifically, we look into \gls{lda} since it is currently the most used topic model, and is applicable for many situations\cite{lda}.
+\gls{lda} uses Dirichlet distributions to calculate the probability of a word belonging to a given topic and a document containing a given topic.
+Topic modeling does not rank documents, which is why an additional part is necessary if we want to search the documents.
+
+Ranking articles can be done in multiple ways, where PageRank is commonly used\cite{google_pagerank2006}.
+PageRank\cite{pagerank_1999} is a ranking algorithm, which is based on the random walk process.
+The principle is to let a number of surfers walk between nodes in a graph, where each surfer takes one step in each iteration.
+After an appropriate number of steps, the number of surfers on each node will be the ranking of the given node, where more surfers give a higher rank.
+The edges in the graph can be based on many interactions, such as hyperlinks, similarities, genres, etc.
+
+In \cite{yang2009topic} \citeauthor{yang2009topic} investigate how to combine topic model \gls{lda}, \gls{lm}, and PageRank.
+They also investigate a modified version of the PageRank algorithm which takes topic-level information into account.
+They use a scientific article dataset where they manually label 200 articles from their dataset based on related queries. 
+In this paper, we want to expand the framework which \cite{yang2009topic} have provided.
+However, the data set which \cite{yang2009topic} uses is based on scientific articles, which includes citation to other similar articles.
+This citation scheme does not necessarily extent to other document types, such as news articles.
+
+In this paper, we want to investigate whether using a similarity measure based on the topics generated by a topic model can be used with PageRank.
+
+The combination of \gls{lda} and PageRank is interesting, since underlying document topics can yield more information than usually given by a single document.
+These topics can be seen as a set of documents sharing a specific topic. 
+
+When evaluating search engines, manual labeling of documents is usually used to measure the performance of models\cite{yang2009topic}\cite{Tang2008}.
+Since the topics, that are created by the \gls{lda}, are distributions of topics in documents, we can use these to evaluate a search algorithm's retrieval performance. 
+
+Our goal is thus to improve news article search by using a combination of \gls{lda}, \gls{lm}, and PageRank.
+For our experiments we use the Nordjyske database of news articles as our dataset.
+We also want to investigate whether an automatic evaluation measure can be used to evaluate a search engine.
+Here we want to look into whether it makes sense to make an evaluation metric based on whether search results are in a given topic.
+Finally, since the news article dataset does not include connections between the articles, how PageRank can be run on this needs to be investigated.
```

Comment:
```suggestion
Finally, since our news article dataset does not include connections between the articles, we want to investigate how we can build an adjacency matrix based on the documents themselves, to be able to run PageRank.
```